core/test/scripts/trace-fixtures/update-fixtures.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// https://docs.google.com/spreadsheets/d/1AaYzpzWnpXQ4JB5IZzOkTO9Zf5Sm0I8dsp7MhBgthMg/edit?usp=sharing
+
+import * as puppeteer from 'puppeteer';
+
+import {LH_ROOT} from '../../../../root.js';
+import {Server} from '../../../../cli/test/fixtures/static-server.js';
+import {saveTrace, saveDevtoolsLog} from '../../../../core/lib/asset-saver.js';
+import collectMetaAnimation from './animation/user-flow.js';
+import collectMetaUserRedirect from './redirect/user-flow.js';
+import collectMetaUserTimings from './user-timings/user-flow.js';
+import collectMetaUserVideoEmbed from './video-embed/user-flow.js';
```

Comment:
> Are you anticipating that more assets than just a single script are necessary to generate some of these artifacts?

Yes, that could happen.

Initially I did it because I was going to put the test fixtures here too, but then realized might as well keep all that in the static server fixtures folder.

Reply:
nit personal opinion: This is going to contaminate the search results in vscode when I look for `user-flow.js`. I don't really see a reason for us to add multiple files in each directory so I would prefer just having them as individual files.